src/Administration/Resources/app/administration/src/module/sw-settings-number-range/page/sw-settings-number-range-list/index.js

```diff
@@ -108,6 +108,10 @@ export default {
             this.showDeleteModal = id;
         },
 
+        canDelete(item) {
+            return !item.type.global;
```

Comment:
we should catch this also on api side 🤔  maybe

Reply:
I was thinking about that too, but if they use API, they could also add it again so the issue is not really present there.
This change is more to avoid accidental deletion, and then not being able to be fixed.

Reply:
Hi @runelaenen 
The ticket is expected to support the merchant in creating a new number range configuration if they have deleted the existing one. In my opinion, we don't have any reason to block the merchant from deleting their global product number range. Please consider closing the pull request or providing an alternative solution. Thank you!